.github/workflows/tests.yml

```diff
@@ -16,8 +20,8 @@ jobs:
       matrix:
         os: [ubuntu-latest]
         python-version: ["3.9", "3.10", "3.11", "3.12"]
-        sphinx: [">=5,<9"]  # Newest Sphinx (any)
-        myst-parser: [">=1,<3"]  # Newest MyST Parser (any)
+        sphinx: [""]  # Newest Sphinx (any)
+        myst-parser: [""]  # Newest MyST Parser (any)
```

Comment:
I think I'd prefer to revert this one change, so that PRs only fail if (within reason) the changes themselves cause a breakage. If we pin our deps to known-good versions, that will help there.

Reply:
Thus the importance of the devdeps run, e.g. no PRs will run into surprises as any issues will be picked up sooner and if needed your pins can be introduced when needed and but not as a blanket measure.

Reply:
@agoose77 - are you OK with my response here, namely that we should not run into any issues when a sphinx release is problematic as we have already seen those with the devdeps testing (see below), and if a problem is complicated to fix, we can introduce a pin specific to that as opposed to this blanket pin.